### ISSUE TITLE
Japanese characters shown as (cid:3821) etc.

### ISSUE THREAD

**Issue by mkudamatsu** [pdfminer/pdfminer.six#130]
I've installed pdfminer.six by `pip install pdfminer.six` into my MacOS Sierra 10.12.5, and successfully converted a PDF file in Japanese into a text file, except that all Japanese characters are shown as (cid:3821) etc.

To solve this issue, I followed what's written at https://github.com/euske/pdfminer/issues/39

However, typing `make cmap` in Terminal from the directory `/usr/local/lib/python2.7/site-packages/pdfminer` returns an error message :
> make: Nothing to be done for `cmap'.

I suspect that it's due to the absence of [`Makefile`](https://github.com/pdfminer/pdfminer.six/blob/master/pdfminer/Makefile) in the `pdfminer` directory. 

What should I do?

**Comment by pietermarsman** [comment]
I'm having trouble replicating this issue with the latest version of pdfminer.six. I've tested it with the PDF from https://github.com/euske/pdfminer/issues/39.

```commandline
$ python .virtualenvs/py37/bin/pdf2txt.py ~/Downloads/h25yosan2.pdf --page-numbers 1 | head -n10
平成２５年予算政府案に盛り込まれた地域活性化施策

施策名

予算額
(百万円)

支援対象

概　要
```

@mkudamatsu if this is still important to you, could you share the pdf that you have trouble with?

**Comment by pietermarsman** [comment]
Closing this because of inactivity and not reproducible.

**Comment by sangwook236** [comment]
Hello!

I have some trouble reading Korean PDF files
I got (cid:~)'s instead of seeing Korean characters.
Below I wrote some information about my test.
I also got the same results when using the APIs of pdfminer library directly in Python.

Thanks in advance.
Sang-Wook

(1) Conda environment:
~/work/testbed/python$ conda list
// packages in environment at ~/anaconda3/envs/pdf:
//
// Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
blas                      1.0                    openblas  
ca-certificates           2021.1.19            h06a4308_1  
certifi                   2020.12.5        py39h06a4308_0  
cffi                      1.14.5                   pypi_0    pypi
chardet                   4.0.0                    pypi_0    pypi
cryptography              3.4.7                    pypi_0    pypi
cycler                    0.10.0           py39h06a4308_0  
dbus                      1.13.18              hb2f20db_0  
expat                     2.3.0                h2531618_2  
fontconfig                2.13.1               h6c09931_0  
freetype                  2.10.4               h5ab3b9f_0  
glib                      2.68.0               h36276a3_0  
gst-plugins-base          1.14.0               h8213a91_2  
gstreamer                 1.14.0               h28cd5cc_2  
icu                       58.2                 he6710b0_3  
jpeg                      9b                   h024ee3a_2  
kiwisolver                1.3.1            py39h2531618_0  
lcms2                     2.12                 h3be6417_0  
ld_impl_linux-64          2.33.1               h53a641e_7  
libffi                    3.3                  he6710b0_2  
libgcc-ng                 9.1.0                hdf63c60_0  
libgfortran-ng            7.3.0                hdf63c60_0  
libopenblas               0.3.13               h4367d64_0  
libpng                    1.6.37               hbc83047_0  
libstdcxx-ng              9.1.0                hdf63c60_0  
libtiff                   4.1.0                h2733197_1  
libuuid                   1.0.3                h1bed415_2  
libxcb                    1.14                 h7b6447c_0  
libxml2                   2.9.10               hb55368b_3  
lz4-c                     1.9.3                h2531618_0  
matplotlib                3.3.4            py39h06a4308_0  
matplotlib-base           3.3.4            py39h62a2d02_0  
ncurses                   6.2                  he6710b0_1  
numpy                     1.19.2           py39h87658db_0  
numpy-base                1.19.2           py39h0f7b65f_0  
olefile                   0.46                       py_0  
openssl                   1.1.1k               h27cfd23_0  
pandas                    1.2.3            py39ha9443f7_0  
pcre                      8.44                 he6710b0_0  
pdf2image                 1.14.0                   pypi_0    pypi
pdfminer-six              20201018                 pypi_0    pypi
pillow                    8.2.0            py39he98fc37_0  
pip                       21.0.1           py39h06a4308_0  
pycparser                 2.20                     pypi_0    pypi
pycryptodome              3.10.1                   pypi_0    pypi
pymupdf                   1.18.10                  pypi_0    pypi
pyparsing                 2.4.7              pyhd3eb1b0_0  
pypdf2                    1.26.0                   pypi_0    pypi
pyqt                      5.9.2            py39h2531618_6  
python                    3.9.2                hdb3f193_0  
python-dateutil           2.8.1              pyhd3eb1b0_0  
pytz                      2021.1             pyhd3eb1b0_0  
qt                        5.9.7                h5867ecd_1  
readline                  8.1                  h27cfd23_0  
scipy                     1.6.2            py39hf56f3a7_0  
setuptools                52.0.0           py39h06a4308_0  
sip                       4.19.13          py39h2531618_0  
six                       1.15.0           py39h06a4308_0  
sortedcontainers          2.3.0                    pypi_0    pypi
sqlite                    3.35.4               hdfb4753_0  
tk                        8.6.10               hbc83047_0  
tornado                   6.1              py39h27cfd23_0  
tzdata                    2020f                h52ac0ba_0  
wand                      0.6.6                    pypi_0    pypi
wheel                     0.36.2             pyhd3eb1b0_0  
xz                        5.2.5                h7b6447c_0  
zlib                      1.2.11               h7b6447c_3  
zstd                      1.4.9                haebb681_0  

(2) Command.
python pdf2txt.py test.pdf --page-numbers 1

(3) PDF file.
[test.pdf](https://drive.google.com/file/d/1kbmNv6umWcuGvzmYv5zequ0OBGpsTg-y/view?usp=sharing)

(4) Result.
(cid:3296)(cid:1204)(cid:2628)(cid:1) (cid:2689)(cid:1576)(cid:1104)(cid:3332)(cid:1) (cid:1175)(cid:2689)(cid:2540)(cid:1) (cid:2001)(cid:3313)(cid:1) (cid:9)(cid:1994)(cid:1088)(cid:2190)(cid:10)

(cid:18)(cid:26)(cid:26)(cid:24)(cid:1433)(cid:1) (cid:2633)(cid:3377)(cid:2628)(cid:1) (cid:2555)(cid:3356)(cid:2596)(cid:1245)(cid:1843)(cid:1) (cid:1157)(cid:1245)(cid:1789)(cid:1) (cid:2571)(cid:1851)(cid:1) (cid:2190)(cid:3365)(cid:2496)(cid:1) (cid:2654)(cid:2050)(cid:3303)(cid:1) (cid:2643)(cid:1549)(cid:1) (cid:1862)(cid:2615)(cid:1) (cid:1945)(cid:2689)(cid:1628)(cid:2633)(cid:1)

(cid:1442)(cid:2937)(cid:1594)(cid:1162)(cid:1) (cid:2643)(cid:1518)(cid:15)(cid:1) (cid:1234)(cid:1) (cid:1586)(cid:2441)(cid:1) (cid:2239)(cid:2658)(cid:2616)(cid:1) (cid:2596)(cid:3303)(cid:1) (cid:1559)(cid:2479)(cid:1601)(cid:1162)(cid:1) (cid:1975)(cid:1814)(cid:2479)(cid:1) (cid:2547)(cid:1549)(cid:1) (cid:2636)(cid:1628)(cid:2633)(cid:1) (cid:3296)(cid:1281)(cid:2019)(cid:2496)(cid:1)

(cid:3252)(cid:1910)(cid:3354)(cid:1594)(cid:1910)(cid:2226)(cid:1) (cid:1155)(cid:2689)(cid:2170)(cid:1861)(cid:1) (cid:2439)(cid:1508)(cid:1732)(cid:1) (cid:2190)(cid:3365)(cid:1) (cid:1087)(cid:1)(cid:2073)(cid:2464)(cid:2496)(cid:1) (cid:1536)(cid:3296)(cid:1)(cid:1203)(cid:2705)(cid:2680)(cid:1) (cid:1945)(cid:2689)(cid:1628)(cid:2633)(cid:1) (cid:2689)(cid:1245)(cid:1594)(cid:1162)(cid:1)

(cid:2643)(cid:1495)(cid:1) (cid:1130)(cid:2633)(cid:1518)(cid:15)(cid:1)

(cid:1234)(cid:1758)(cid:1377)(cid:1) (cid:2687)(cid:2876)(cid:1) (cid:1528)(cid:1532)(cid:2647)(cid:1628)(cid:1576)(cid:1) (cid:2571)(cid:1851)(cid:1) (cid:2190)(cid:3365)(cid:1086)(cid:1) (cid:2479)(cid:1493)(cid:1) (cid:1249)(cid:2616)(cid:1) (cid:2230)(cid:3094)(cid:3303)(cid:2464)(cid:1) (cid:3297)(cid:2769)(cid:13)(cid:1) (cid:1234)(cid:1851)(cid:1162)(cid:1) (cid:2479)

(cid:1676)(cid:1137)(cid:1) (cid:1086)(cid:2464)(cid:1) (cid:3297)(cid:2769)(cid:2496)(cid:1) (cid:1536)(cid:3303)(cid:1) (cid:1914)(cid:2004)(cid:3296)(cid:1) (cid:3303)(cid:2024)(cid:2616)(cid:1) (cid:2874)(cid:2769)(cid:1) (cid:1925)(cid:3294)(cid:1162)(cid:1) (cid:2643)(cid:1495)(cid:1) (cid:1632)(cid:3294)(cid:1518)(cid:15)(cid:1) (cid:1234)(cid:1) (cid:1150)(cid:1175)(cid:1) (cid:2633)(cid:3303)(cid:1)

(cid:1532)(cid:2190)(cid:2647)(cid:1628)(cid:1088)(cid:2628)(cid:1) (cid:2876)(cid:2640)(cid:1173)(cid:2001)(cid:2633)(cid:1) (cid:1360)(cid:2633)(cid:2769)(cid:1) (cid:2443)(cid:1162)(cid:1) (cid:2643)(cid:2618)(cid:2616)(cid:1) (cid:1918)(cid:1576)(cid:3294)(cid:1137)(cid:1) (cid:1595)(cid:1518)(cid:15)(cid:1) (cid:1234)(cid:1758)(cid:1377)(cid:1) (cid:2190)(cid:3365)(cid:1203)(cid:2239)

(cid:2583)(cid:2615)(cid:1) (cid:2689)(cid:1576)(cid:2680)(cid:2613)(cid:1789)(cid:1) (cid:2737)(cid:2479)(cid:2771)(cid:1) (cid:3357)(cid:1586)(cid:1173)(cid:1088)(cid:1) (cid:1394)(cid:2496)(cid:2226)(cid:1) (cid:2647)(cid:2344)(cid:2628)(cid:1) (cid:2633)(cid:2634)(cid:2616)(cid:1) (cid:1235)(cid:1536)(cid:3354)(cid:3294)(cid:1776)(cid:1495)(cid:1) (cid:2707)(cid:2660)(cid:2640)

(cid:2616)(cid:1)(cid:1093)(cid:2441)(cid:3294)(cid:1910)(cid:1) (cid:1087)(cid:1)(cid:1104)(cid:2635)(cid:2628)(cid:1) (cid:3311)(cid:1586)(cid:2001)(cid:2343)(cid:2616)(cid:1) (cid:2636)(cid:2687)(cid:3296)(cid:1576)(cid:1) (cid:1394)(cid:1789)(cid:1) (cid:2689)(cid:3296)(cid:3294)(cid:1122)(cid:1377)(cid:1) (cid:2604)(cid:1576)(cid:3294)(cid:2769)(cid:1) (cid:1925)(cid:3310)(cid:1518)

(cid:1495)(cid:1) (cid:2959)(cid:1910)(cid:2496)(cid:2226)(cid:1) (cid:2190)(cid:3365)(cid:2680)(cid:1) (cid:1945)(cid:2689)(cid:1734)(cid:1) (cid:116)(cid:3289)(cid:2563)(cid:3296)(cid:1) (cid:2689)(cid:1576)(cid:2680)(cid:1) (cid:2635)(cid:3280)(cid:1732)(cid:1843)(cid:1) (cid:2689)(cid:1656)(cid:2496)(cid:1) (cid:1173)(cid:1241)(cid:3294)(cid:2769)(cid:1) (cid:1925)(cid:3294)(cid:2504)(cid:1)

(cid:1994)(cid:2212)(cid:3296)(cid:1) (cid:2689)(cid:1576)(cid:2680)(cid:1) (cid:2346)(cid:3201)(cid:117)(cid:1789)(cid:1) (cid:1088)(cid:2737)(cid:1596)(cid:1) (cid:2299)(cid:1) (cid:2643)(cid:1518)(cid:15)(cid:1)

(cid:2689)(cid:1576)(cid:2680)(cid:2635)(cid:1) (cid:2346)(cid:3201)(cid:1495)(cid:1) (cid:1234)(cid:1) (cid:2583)(cid:2635)(cid:2616)(cid:1) (cid:2689)(cid:1576)(cid:2628)(cid:1) (cid:1175)(cid:2644)(cid:13)(cid:1) (cid:2689)(cid:1576)(cid:2628)(cid:1) (cid:2071)(cid:2706)(cid:13)(cid:1) (cid:2071)(cid:2680)(cid:2682)(cid:3296)(cid:1) (cid:2689)(cid:1576)(cid:1) (cid:1633)(cid:1)

(cid:2241)(cid:1) (cid:1086)(cid:2769)(cid:1) (cid:3339)(cid:3093)(cid:2496)(cid:2226)(cid:1) (cid:2874)(cid:2616)(cid:1) (cid:2299)(cid:1) (cid:2643)(cid:1518)(cid:15)(cid:1) (cid:2689)(cid:1576)(cid:2628)(cid:1) (cid:1175)(cid:2644)(cid:2615)(cid:1) (cid:1175)(cid:1518)(cid:1231)(cid:2689)(cid:1732)(cid:1495)(cid:1) (cid:3339)(cid:3093)(cid:1789)(cid:1) (cid:1377)(cid:3083)(cid:1377)(cid:1908)(cid:1)

(cid:2689)(cid:1576)(cid:2628)(cid:1) (cid:2071)(cid:2706)(cid:2615)(cid:1) (cid:1942)(cid:2773)(cid:2226)(cid:1789)(cid:1) (cid:3252)(cid:3333)(cid:1594)(cid:1495)(cid:1) (cid:1155)(cid:2571)(cid:1086)(cid:1) (cid:1862)(cid:1518)(cid:15)(cid:1) (cid:2689)(cid:1576)(cid:2628)(cid:1) (cid:1175)(cid:2075)(cid:1241)(cid:9)(cid:3812)(cid:5109)(cid:4052)(cid:10)(cid:2628)(cid:1) (cid:1945)(cid:2689)

(cid:1086)(cid:1) (cid:2439)(cid:1508)(cid:1732)(cid:1) (cid:1245)(cid:1536)(cid:3310)(cid:1549)(cid:1) (cid:1918)(cid:3252)(cid:2540)(cid:1) (cid:1518)(cid:1842)(cid:1) (cid:3372)(cid:1175)(cid:1843)(cid:1) (cid:1994)(cid:2212)(cid:3294)(cid:1495)(cid:1) (cid:1155)(cid:2571)(cid:1) (cid:3303)(cid:1532)(cid:1) (cid:2689)(cid:1576)(cid:1495)(cid:1) (cid:2071)(cid:2680)(cid:2682)(cid:3296)(cid:1)

(cid:1130)(cid:2613)(cid:1789)(cid:1) (cid:1088)(cid:2737)(cid:1595)(cid:1518)(cid:15)(cid:1) (cid:2689)(cid:1576)(cid:2540)(cid:1) (cid:1177)(cid:1778)(cid:3294)(cid:2504)(cid:1) (cid:2226)(cid:1203)(cid:2190)(cid:3365)(cid:2496)(cid:2226)(cid:1495)(cid:1) (cid:2689)(cid:1576)(cid:1843)(cid:1) (cid:2925)(cid:1536)(cid:3296)(cid:1) (cid:2687)(cid:2966)(cid:9)(cid:6742)(cid:7352)(cid:10)(cid:3294)

(cid:1137)(cid:1) (cid:1203)(cid:2239)(cid:3294)(cid:2504)(cid:1) (cid:1245)(cid:1536)(cid:2966)(cid:2496)(cid:1) (cid:1086)(cid:2658)(cid:1) (cid:1236)(cid:2190)(cid:3296)(cid:1) (cid:3372)(cid:1175)(cid:1843)(cid:1) (cid:2483)(cid:2613)(cid:1776)(cid:1) (cid:3296)(cid:1) (cid:1992)(cid:1910)(cid:13)(cid:1) (cid:3296)(cid:1204)(cid:2190)(cid:3365)(cid:2496)(cid:2226)(cid:1495)(cid:1) (cid:2689)

(cid:1576)(cid:2496)(cid:1) (cid:2573)(cid:2570)(cid:2628)(cid:1) (cid:1937)(cid:1843)(cid:1) (cid:2195)(cid:1854)(cid:1) (cid:2299)(cid:1) (cid:2643)(cid:1495)(cid:1) (cid:2023)(cid:2596)(cid:1843)(cid:1) (cid:2106)(cid:1198)(cid:2680)(cid:1) (cid:1415)(cid:1137)(cid:1)(cid:2689)(cid:1173)(cid:3294)(cid:2504)(cid:1) (cid:2573)(cid:2570)(cid:3294)(cid:1495)(cid:1) (cid:2190)(cid:1736)(cid:2628)(cid:1)

(cid:1502)(cid:1777)(cid:2496)(cid:1) (cid:1546)(cid:1) (cid:1862)(cid:2615)(cid:1) (cid:1086)(cid:2746)(cid:2966)(cid:1843)(cid:1) (cid:1601)(cid:2479)(cid:2529)(cid:1) (cid:1155)(cid:3313)(cid:2633)(cid:1) (cid:2643)(cid:1518)(cid:15)(cid:1) (cid:1234)(cid:1) (cid:1150)(cid:1175)(cid:1) (cid:1945)(cid:2689)(cid:1086)(cid:1) (cid:1994)(cid:2212)(cid:3294)(cid:1910)(cid:1) (cid:2689)(cid:1576)

(cid:2049)(cid:1518)(cid:1495)(cid:1) (cid:2190)(cid:1736)(cid:2496)(cid:1137)(cid:2226)(cid:1) (cid:1890)(cid:2679)(cid:1) (cid:1234)(cid:1)(cid:2583)(cid:2635)(cid:2616)(cid:1) (cid:2874)(cid:1495)(cid:1) (cid:2472)(cid:2202)(cid:2616)(cid:1) (cid:2049)(cid:2504)(cid:2547)(cid:1518)(cid:15)

(cid:1155)(cid:2689)(cid:3295)(cid:2615)(cid:1) (cid:2633)(cid:1758)(cid:3296)(cid:1) (cid:3296)(cid:1204)(cid:2680)(cid:1) (cid:2685)(cid:1236)(cid:2001)(cid:2343)(cid:2496)(cid:1) (cid:2633)(cid:2628)(cid:1843)(cid:1) (cid:2689)(cid:1245)(cid:3296)(cid:1518)(cid:15)(cid:1) (cid:2549)(cid:1403)(cid:3294)(cid:1910)(cid:1) (cid:2635)(cid:1088)(cid:2628)(cid:1) (cid:3300)

(cid:1851)(cid:2239)(cid:9)(cid:83)(cid:66)(cid:85)(cid:74)(cid:80)(cid:79)(cid:66)(cid:77)(cid:74)(cid:85)(cid:90)(cid:10)(cid:2616)(cid:1) (cid:116)(cid:2647)(cid:1245)(cid:1) (cid:2633)(cid:2634)(cid:2616)(cid:1) (cid:1235)(cid:1536)(cid:3354)(cid:3294)(cid:1776)(cid:1495)(cid:1) (cid:1155)(cid:3313)(cid:117)(cid:2613)(cid:1789)(cid:1) (cid:1231)(cid:2687)(cid:3294)(cid:1162)(cid:1) (cid:2643)(cid:1245)(cid:1) (cid:1656)(cid:1945)

(cid:2633)(cid:1518)(cid:15)(cid:1) (cid:2633)(cid:1) (cid:1155)(cid:2571)(cid:1) (cid:1104)(cid:2635)(cid:2633)(cid:1) (cid:2681)(cid:2899)(cid:1843)(cid:1) (cid:2596)(cid:3303)(cid:1) (cid:2636)(cid:3297)(cid:1) (cid:1130)(cid:2633)(cid:1732)(cid:1495)(cid:1) (cid:1245)(cid:1536)(cid:1495)(cid:1) (cid:1234)(cid:2496)(cid:1) (cid:2202)(cid:2621)(cid:3294)(cid:1495)(cid:1) (cid:1992)(cid:1536)

(cid:1241)(cid:2071)(cid:1086)(cid:1) (cid:2643)(cid:2616)(cid:1) (cid:1656)(cid:1861)(cid:1) (cid:1086)(cid:1502)(cid:3296)(cid:1) (cid:1086)(cid:2687)(cid:2633)(cid:1518)(cid:15)(cid:1) (cid:1947)(cid:1791)(cid:1) (cid:3372)(cid:2570)(cid:2633)(cid:1732)(cid:1495)(cid:1) (cid:1104)(cid:1435)(cid:2613)(cid:1789)(cid:1) (cid:2633)(cid:2634)(cid:2633)(cid:1732)(cid:1495)(cid:1) (cid:2570)

(cid:2479)(cid:1843)(cid:1) (cid:1536)(cid:2899)(cid:3297)(cid:1) (cid:1155)(cid:2571)(cid:1) (cid:2633)(cid:1758)(cid:3296)(cid:1) (cid:1086)(cid:2687)(cid:2615)(cid:1) (cid:2542)(cid:3354)(cid:1596)(cid:1) (cid:2299)(cid:1) (cid:2643)(cid:2613)(cid:1377)(cid:1) (cid:1245)(cid:2052)(cid:2680)(cid:2613)(cid:1789)(cid:1) (cid:2635)(cid:1088)(cid:2615)(cid:1) (cid:2333)(cid:2333)(cid:1789)

(cid:1843)(cid:1) (cid:2596)(cid:3303)(cid:1) (cid:2636)(cid:3294)(cid:1495)(cid:1) (cid:2707)(cid:2660)(cid:1732)(cid:1495)(cid:1) (cid:2583)(cid:2967)(cid:2496)(cid:1495)(cid:1) (cid:2040)(cid:3354)(cid:1086)(cid:1) (cid:2489)(cid:1518)(cid:15)(cid:1) (cid:2633)(cid:1) (cid:1155)(cid:2571)(cid:1) (cid:2689)(cid:1576)(cid:1843)(cid:1) (cid:3134)(cid:3303)(cid:1) (cid:2333)(cid:2333)(cid:1789)

(cid:1843)(cid:1) (cid:2596)(cid:3303)(cid:1) (cid:2636)(cid:3294)(cid:1495)(cid:1) (cid:1130)(cid:2633)(cid:1) (cid:2190)(cid:3365)(cid:2680)(cid:2613)(cid:1789)(cid:1576)(cid:1) (cid:1245)(cid:2504)(cid:3297)(cid:1) (cid:2299)(cid:1) (cid:2643)(cid:1576)(cid:1790)(cid:1) (cid:2689)(cid:1576)(cid:1843)(cid:1) (cid:2687)(cid:1979)(cid:3294)(cid:1137)(cid:1) (cid:1203)(cid:2239)(cid:3294)

(cid:2769)(cid:1) (cid:2443)(cid:2613)(cid:1910)(cid:1) (cid:2689)(cid:1576)(cid:2628)(cid:1) (cid:2346)(cid:3201)(cid:1495)(cid:1) (cid:2520)(cid:2687)(cid:1594)(cid:2479)(cid:1) (cid:2643)(cid:1495)(cid:1) (cid:2636)(cid:2633)(cid:1518)(cid:15)(cid:1) (cid:1234)(cid:1758)(cid:1970)(cid:1789)(cid:1) (cid:2571)(cid:1851)(cid:2190)(cid:3365)(cid:1) (cid:1087)(cid:1) (cid:2073)(cid:2464)(cid:2496)(cid:1)

(cid:3303)(cid:1150)(cid:1594)(cid:2769)(cid:1)(cid:2443)(cid:1162)(cid:1) (cid:2643)(cid:1495)(cid:1) (cid:2504)(cid:1758)(cid:1) (cid:1086)(cid:2769)(cid:1) (cid:1162)(cid:2773)(cid:2680)(cid:2635)(cid:1) (cid:1945)(cid:2689)(cid:1495)(cid:1) (cid:1104)(cid:2635)(cid:2680)(cid:1) (cid:2864)(cid:2583)(cid:2496)(cid:2226)(cid:1861)(cid:1) (cid:3303)(cid:1150)(cid:2001)(cid:2441)(cid:2616)(cid:1)

(cid:2874)(cid:2616)(cid:1) (cid:1130)(cid:2633)(cid:1) (cid:2439)(cid:1508)(cid:1732)(cid:1) (cid:2689)(cid:1576)(cid:2680)(cid:2635)(cid:1)(cid:2959)(cid:1910)(cid:2496)(cid:2226)(cid:1) (cid:3303)(cid:1150)(cid:2876)(cid:2616)(cid:1) (cid:1917)(cid:2205)(cid:3303)(cid:2464)(cid:1) (cid:3297)(cid:1) (cid:3289)(cid:2563)(cid:1086)(cid:1) (cid:2643)(cid:1518)(cid:15)(cid:1)

(cid:2052)(cid:2583)(cid:2496)(cid:2226)(cid:1495)(cid:1) (cid:2633)(cid:1975)(cid:1) (cid:2299)(cid:1433)(cid:1) (cid:2681)(cid:2496)(cid:1) (cid:2571)(cid:1851)(cid:1) (cid:1155)(cid:2689)(cid:1) (cid:1986)(cid:1) (cid:2190)(cid:3365)(cid:2628)(cid:1) (cid:2737)(cid:2563)(cid:1945)(cid:2689)(cid:1628)(cid:2616)(cid:1) (cid:2689)(cid:1576)(cid:2680)(cid:2635)(cid:1)

(cid:14)(cid:1) (cid:18)(cid:1) (cid:14)
